client/src/views/Home/HomePage/HomePage.tsx

```diff
@@ -1,40 +1,43 @@
+import React, { useState } from 'react';
 import { Box, Grid, Typography } from '@mui/material';
 import { SwiperSlide } from 'swiper/react';
-import React, { useState } from 'react';
-import DiscountCard from '../../../components/card/DiscountCard/DiscountCard';
+import DiscountCard from 'components/card/DiscountCard/DiscountCard';
```

Comment:
create and index.ts file and export all files in components folder. 
for simple components, that are not pages, use simple 'export' not the 'export default'
you'll be able to import all of the files import {DiscountCard, PersonalCard ...} from 'components';